### PR TITLE
Update readme and routing

### PR DESCRIPTION
**Pull request by ledn-reviewer**

### CODE REVIEW

File: src/Router.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+// Note: This is just to help set you up more quickly, feel free to adjust if needed.
+export function Router() {
+  return (
+    <Routes>
+      <Route index path="/" element={<List />} />
+      <Route path="/:id" element={<Details />} />
+    </Routes>
+  );
+}
```
Comment (nathan-ledn) [inline src/Router.tsx]: Jumpline or not needed?

File: src/pages/List/index.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
   const { data: detail } = usePlanetDetail('1');
 
   console.log('list', list);
   console.log('detail', detail);
-  return null;
+  return <>List</>;
 };
 
-export default DataExample;
+export default List;
```
Comment (nathan-ledn) [inline src/pages/List/index.tsx]: Jumpline or not needed?

**Review by nathan-ledn — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -59,4 +59,12 @@ To ensure the success of your mission, adhere to the following technical specifi
...
-* Provide screenshots or recording of the final implementation.
\ No newline at end of file
+* Provide screenshots or recording of the final implementation.
+* In order to allow us to review a similar set of challenges, please do not change anything from the server.
+
+
+<div align='center'>
+  <h3>Do not hesitate to ask us any questions if you have any.</h3>
+  <h3>Remember, we want to see the best you would deliver on your day to day.</h3>
+  <h1>May the Force be with you!</h1>
```
Comment (mairabello-ledn) [inline README.md:69]: 😆

**Review by mairabello-ledn — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -44,19 +44,28 @@ In the wake of the Empire's fall, the Republic has turned to Coruscant Bank to a
...
 
 * Organize the UI to the best of your knowledge and judgment.
-  * All available data can be found in the `DataExample` component. 
+    * All available data can be found in the `DataExample` component.
 * Do not use any existing UI library or framework (nothing like Tailwind, Bootstrap or Material UI for instance).
-  * Reset / normalize stylesheets are allowed. 
-* Provide screenshots or recording of the final implementation.
\ No newline at end of file
+    * Reset / normalize stylesheets are allowed.
+* The interface must be responsive.
```
Comment (mairabello-ledn) [inline README.md]: I feel like this is not a technical requirement, it's a functional requirement. So maybe put it together with the ones in the section before?

Comment (emilie-t-ledn) [reply README.md]: Good point. Fixed it

File: README.md [hunk 5]
```diff
@@ -41,22 +41,30 @@ In the wake of the Empire's fall, the Republic has turned to Coruscant Bank to a
 
 ## Requirements
 
-Your task is to design the foundation of a user interface capable of identifying planets susceptible to uprisings and enabling proactive intervention. The objective is to build a UI that assists administrators in analyzing transactions across various planets. To meet this goal, adhere to the following guidelines:
+Your task is to design the foundation of a responsive user interface capable of identifying planets susceptible to uprisings and enabling proactive intervention. The objective is to build a UI that assists administrators in analyzing transactions across various planets. To meet this goal, adhere to the following guidelines:
```
Comment (mairabello-ledn) [inline README.md:44]: I like it! It's not as clear as a bullet point, but it shouldn't need to be, just a small reminder.

**Pull request merged**